apps/server/dist/apps/server/src/lib/mongoose/models/Report.js

```diff
@@ -18,7 +18,7 @@ const ReviewSchema = new mongoose_1.Schema({
 //* timestamps automatically creates createdAt and updatedAt properties
```

Comment:
I'm a little concerned that the `dist` folder is showing up in commits. I think we need to add it to `.gitignore`?